source/app/scorearea.cpp

```diff
@@ -354,3 +357,12 @@ ScoreArea::loadTheme(const SettingsManager &settings_manager, bool redraw)
         myDisableRedraw = false;
     }
 }
+
+void
+ScoreArea::loadSystemSpacing(const SettingsManager &settings_manager, bool redraw)
+{
+  auto settings = settings_manager.getReadHandle();
```

Comment:
It looks like there's only a 2-space indent here, but it should be 4

Reply:
Got it! Should be fixed now